neuraxle/data_container.py

```diff
@@ -221,8 +386,13 @@ class ListDataContainer(DataContainer):
     """
 
     @staticmethod
-    def empty() -> 'ListDataContainer':
-        return ListDataContainer([], [], [])
+    def empty(original_data_container: DataContainer = None) -> 'ListDataContainer':
+        if original_data_container is None:
+            sub_data_containers = []
+        else:
```

Comment:
I find this really weird to pass `original_data_container`, I don't understand why we'd do that with that passed object in a method named `empty`.

Reply:
resolved to an already existing issue https://github.com/Neuraxio/Neuraxle/issues/175  this would take too long to fix. the template methods I have talked to you about might work : _will_validate. stay tuned.